app/segments/drill/[id]/submission/input.js

```diff
@@ -428,8 +438,7 @@ function invalidateOnSubmit(queryClient, drillId, teamId, userId) {
 export default function Input({ drillInfo, setToggleResult, setOutputData }) {
   //Helper varibles
   const { id, assignedTime } = useLocalSearchParams();
-
-  console.log("ID AND THE ASSIGNED TIME", id, assignedTime);
+  const insets = useContext(SafeAreaInsetsContext);
```

Comment:
What's this?

Reply:
for the BottomSheetBackdrop (the darkened background when you open the modal), I'm shifting it up by the status bar amount because it didn't extend up there originally, probably due to the Safe Areas. Insets is the cleanest way to get the status bar height on all platforms that I can find